docs/garage.md

```diff
@@ -0,0 +1,394 @@
+# Using Garage Object Storage in Bento
+
+This document covers how to use [Garage](https://garagehq.deuxfleurs.fr/), a lightweight, self-hosted, S3-compatible object storage solution, as an alternative to MinIO in your Bento deployment.
```

Comment:
"as an alternative to MinIO in your Bento deployment" --> "which replaces Minio in Bento deployments"